### ISSUE TITLE
Document accessibility issues with donut chart

### ISSUE THREAD

**Issue by benjiallen** [benjiallen/accessible-python-dev-survey-2020#1]
Review the donut chart (see Figure 1) used within the [python developers survey](https://www.jetbrains.com/lp/python-developers-survey-2020/). See the section entitled "Do you consider yourself a Data Scientist?".

Figure 1
<img width="429" alt="2021-04-09_20-48-36" src="https://user-images.githubusercontent.com/16546334/114254410-32f7e280-997d-11eb-8c07-2d4678d5d72a.png" alt="Donut chart used in survey results">

The review should be completed with respect to the WCAG 2.1 level A and AA guidelines.

In addition to the accessibility review, consider these questions:

* Could this visualization be easily represented in a data table?
* Does using a data table make the experience more or less efficient? Would important visual data, like trends, be lost in a data table presentation?
* Would an alternative visualization do a better job of conveying the data? If so, why?
* Does the chart title and supporting text do a good enough job in explaining the data within the visualization?
